roles/variables/vars/main.yml

```diff
@@ -1883,7 +1883,7 @@ kafka_connect_replicator_properties:
   kafka_connect_oauth_scope:
     enabled: "{{kafka_connect_replicator_oauth_enabled and oauth_groups_scope!='none' }}"
```

Comment:
shouldn't the condition also change in that case? we are checking for groups_scope and setting groups_claim?